docs/getting-started-guides/centos/centos_manual_config.md

```diff
@@ -1,236 +0,0 @@
----
-assignees:
-- thockin
-title: CentOS
----
-
```

Comment:
For what it's worth, I think we should post a notice that this document is out of date and deprecated, along with a redirect or a new link that users should follow. I think that's better than simply deleting the file outright. We can still remove it from the table of contents and add an exception to the TOC check, as it is considered deprecated.